src/TestHelper/ref/TestHelper.cs

```diff
@@ -0,0 +1,127 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This doesn't look like it was generated by GenAPI. Any reason not to have it generated so we can easily regenerate in the future?